app/notification_email_test.go

```diff
@@ -89,6 +90,7 @@ func TestGetNotificationEmailBodyFullNotificationPublicChannel(t *testing.T) {
 	teamStoreMock := mocks.TeamStore{}
 	teamStoreMock.On("GetByName", "testteam").Return(&model.Team{Name: "testteam"}, nil)
 	storeMock.On("Team").Return(&teamStoreMock)
+	setupTeamServiceWithMocks(t, th.App, storeMock)
```

Comment:
Can we call this from inside `SetupWithStoreMock` itself? Might help to avoid adding new lines everywhere.

Reply:
Makes sense 👍